enterprise/frontend/src/embedding-sdk/components/public/InteractiveQuestion/InteractiveQuestion.unit.spec.tsx

```diff
@@ -66,8 +66,12 @@ function InteractiveQuestionTestResult() {
 
 const setup = ({
   isValidCard = true,
+  withCustomLayout = false,
+  withChartTypeSelector = true,
```

Comment:
I found using different values for booleans could be confusing, in the test we won't be consciously sure which option is what value.

I propose if you want to have default values, let them be false by default since that's what most optional booleans behave, and only pass `property: true` when needed.

What do you think?

Reply:
@WiNloSt Good eye! That default prop indeed doesn't make sense, this is fixed in https://github.com/metabase/metabase/pull/50664/commits/67693f3281163165985369c01a45f93389f308c6